OpenSim/Common/Component.h

```diff
@@ -2793,9 +2793,7 @@ void Input<T>::connect(const AbstractOutput& output,
             else {
                 pathName = pathName + "/" + chan.second.getName();
             }
-            if (!annotation.empty() && annotation != chan.second.getChannelName()) {
-                pathName += "(" + annotation + ")";
-            }
+            pathName += "(" + alias + ")";
```

Comment:
I think you only want to put in the parentheses if `!alias.empty()`. Is this a change in behavior from before? See my PR review comment on `AbstractInput::parseConnecteeName()`